.tekton/tasks/yaml-lint.yaml

```diff
@@ -1,50 +0,0 @@
-apiVersion: tekton.dev/v1
-kind: Task
-metadata:
-  name: yaml-lint
-  labels:
-    app.kubernetes.io/version: "0.1"
-  annotations:
-    tekton.dev/pipelines.minVersion: "0.12.1"
-    tekton.dev/categories: Code Quality
-    tekton.dev/tags: linter
-    tekton.dev/displayName: "YAML linter"
-    tekton.dev/platforms: "linux/amd64"
-spec:
-  description: >-
-    This task can be used to perform lint check on YAML files
-  workspaces:
-    - name: shared-workspace
-      description: A workspace that contains the fetched git repository.
-  params:
-    - name: args
-      type: array
-      description: extra args needs to append
-      default: ["--help"]
-  steps:
-    - name: lint-yaml-files
-      image: docker.io/cytopia/yamllint:1.26@sha256:1bf8270a671a2e5f2fea8ac2e80164d627e0c5fa083759862bbde80628f942b2  # tag: 1.23
-      # per https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting
-      # the cluster will set imagePullPolicy to IfNotPresent
-      workingDir: $(workspaces.shared-workspace.path)/source
-      command:
-        - yamllint
-      args:
-        - $(params.args)
-    - name: ensure-params-not-in-script
-      image: quay.io/konflux-ci/appstudio-utils:ab6b0b8e40e440158e7288c73aff1cf83a2cc8a9@sha256:24179f0efd06c65d16868c2d7eb82573cce8e43533de6cea14fec3b7446e0b14
-      # per https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting
-      # the cluster will set imagePullPolicy to IfNotPresent
-      script: |
-        #!/bin/bash
-        for task in $(find task -name '*.yaml'); do
-          if yq '.spec?.steps[] | .script' $task | grep -q '\$(params\.'; then
-            FAILED_TASKS="$FAILED_TASKS $task"
-          fi
-        done
-        if [ -n "$FAILED_TASKS" ]; then
-          echo "Tasks contains params in script section (https://tekton.dev/docs/pipelines/tasks/#substituting-in-script-blocks)"
-          echo $FAILED_TASKS | tr ' ' '\n' | sort
-          exit 1
-        fi
-      workingDir: $(workspaces.shared-workspace.path)/source
```

Comment:
The yamllint task also did something entirely not yamllint-y (^ this check)

We need to preserve this check